OpenSim/Moco/RegisterTypes_osimMoco.cpp

```diff
@@ -27,10 +27,11 @@
 #include "MocoFrameDistanceConstraint.h"
 #include "MocoGoal/MocoAccelerationTrackingGoal.h"
 #include "MocoGoal/MocoAngularVelocityTrackingGoal.h"
-#include "MocoGoal/MocoContactTrackingGoal.h"
+#include "MocoGoal/MocoContactImpulseTrackingGoal.h"
```

Comment:
You accidentally renamed the MocoContactTrackingGoal.h include and left MocoImpulseTrackingGoal.h include unchanged below.